custom/icds_reports/tasks.py

```diff
@@ -0,0 +1,23 @@
+import os
+
+from celery.schedules import crontab
+from celery.task import periodic_task
+from django.conf import settings
+
+from django.db import connections
+
+
+@periodic_task(run_every=crontab(minute=0, hour=0, day_of_week=7), acks_late=True)
+def move_ucr_data_into_aggregation_tables():
+    with connections[settings.ICDS_UCR_DATABASE_ALIAS].cursor() as cursor:
```

Comment:
I realized the checking of the setting was actually broken in the UCR. Fixed both here: https://github.com/dimagi/commcare-hq/pull/13079/commits/95e8d50411b3ad738f5e34ca6db8b0ddc5da885d